app/stac_api/widgets/label_widget.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.utils.safestring import mark_safe
+
+
+class LabelWidget(forms.TextInput):
+    """Custom widget to display a label without input"""
+
+    def render(self, name, value, attrs=None, renderer=None):
+        if value:
+            return mark_safe(f'{value}')
```

Comment:
If you do this
`return mark_safe(f'<div class="readonly">{value}</div>')`
then the value will be aligned properly :)